app/src/main/res/values/themes.xml

```diff
@@ -8,6 +8,12 @@
         <!-- Assign dialog style here. -->
         <item name="alertDialogTheme">@style/MyAlertDialogStyle</item>
         <item name="alertDialogStyle">@style/MyAlertDialogStyle</item>
+
+        <!--<item name="appRatingDialogTitleStyle">@style/MyDialogTitleStyle</item>-->
```

Comment:
Can this be deleted?

Reply:
I would like to keep it. We currently have default styles for that, which are used in example. This part of code is not used in example app, but it is shown in documentation.

Reply:
Maybe add a comment then what this is so that's it's clear that this piece of code was left on purpose and what it does?

Reply:
comment added